tools/init_dev.js

```diff
@@ -64,7 +65,17 @@ if (process.argv.length > 2 && process.argv[2] === '-f') {
   force = true;
 }
 
-const futures = [];
+// need to manually delete ts under midway since midway does not handle yarn installation properly
```

Comment:
@WenheLI May I ask you to describe which is not handled properly for yarn, thus we can track which could be improved :)